rollup.config.ts

```diff
@@ -2,20 +2,32 @@ import resolve from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
 import typescript from 'rollup-plugin-typescript2';
 import { terser } from 'rollup-plugin-terser';
+import visualizer from 'rollup-plugin-visualizer';
+
+const commonPlugins = () => [typescript({ useTsconfigDeclarationDir: true, objectHashIgnoreUnknownHack: true, }), commonjs(), resolve()]; 
+
 
 export default [
   {
     input: { zarr: 'src/zarr.ts', core: 'src/zarr-core.ts' },
-    output: {
+    output: [{
       dir: 'dist/',
       format: 'es',
       entryFileNames: '[name].mjs',
       sourcemap: true,
     },
+    {
+      dir: 'dist/',
+      format: 'es',
+      entryFileNames: '[name].min.mjs',
+      sourcemap: true,
+      plugins: [terser()]
+    },
+  ],
     watch: {
       include: 'src/**',
     },
-    plugins: [typescript({ useTsconfigDeclarationDir: true }), commonjs(), resolve()],
+    plugins: [...commonPlugins(), visualizer({filename: "stats.html"}), visualizer({filename: "stats.min.html", sourcemap: true})],
```

Comment:
Can `plugins === commonPlugins()` here and `output[0].plugins === [ visualizer({filename: "stats.html"}) ]` and `output[1].plugins === [ terser(), visualizer({filename: "stats.min.html", sourcemap: true }) ]`

Reply:
The visualizer plugin didn't want to be an output plugin so I had to use this workaround

Reply:
Ok, not a big deal :)